DNN Platform/Modules/ResourceManager/Dnn.Modules.ResourceManager.csproj

```diff
@@ -252,6 +258,18 @@
       <Link>stylecop.json</Link>
```

Comment:
I think the stylecop.json is twice in the csproj file

Reply:
Yes it is, I'll remove the 2nd entry in the next code push